src/index.ts

```diff
@@ -31,6 +34,21 @@ async function assertFileExists(path: string) {
   );
 }
 
+function processBundleBudgets() {
+  // bundles inputs will be dynamic
+  // e.g bundle-{name}: 50
+  const bundleBudgets: BundleBudget[] = [];
+  for (let [k, v] of Object.entries(process.env)) {
+    // all github action inputs have this prefix "INPUT_", the real input starts with bundle-
```

Comment:
**curious** Do they? Is this because you're parsing `process.env` and not using `core.getInput`?

Is there a way we could rely on `core.getInput` instead? That would remove the need to parse user input this way.

Reply:
The reason to parse all the inputs is that we don't know what the bundle inputs are because they are dynamic, this is the same logic that `core.getInput()` uses.

<img width="695" alt="Screen Shot 2022-05-11 at 12 59 10 AM" src="https://user-images.githubusercontent.com/13124163/167779072-fe8a4523-5e6d-4d5b-9c95-179c789973fc.png">

src:  https://github.community/t/how-to-get-all-inputs-from-an-gh-action-without-knowing-input-names/207661/2